std_msgs/Header header
geometry_msgs/Point point
float64 angle

uint8 gesture_id
uint32 image_height
uint32 image_width

uint8 NONE=0
uint8 THUMB_UP=1
uint8 THUMB_DOWN=2
uint8 VICTORY=3
uint8 POINTING_UP=4
uint8 CLOSED_FIST = 5
uint8 OPEN_PALM = 6
uint8 ILOVEU = 7